### PR TITLE
Update to version 1.1.1

### PR DESCRIPTION
**Pull request by marcoesters**
Jira ticket: [PKG-396](https://anaconda.atlassian.net/jira/software/c/projects/PKG/issues/PKG-396)

# Package Info

Home: https://joblib.readthedocs.io/en/latest/
Development: https://github.com/joblib/joblib
Documentation: https://joblib.readthedocs.io/en/latest/
Change log: https://github.com/joblib/joblib/blob/master/CHANGES.rst
Git diff: https://github.com/joblib/joblib/compare/1.1.0...1.1.1

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -21,14 +20,14 @@ requirements:
...
 test:
   requires:
     - pip
-    - python <3.10
-    - pytest
+    - python
+    - pytest  # pytest is needed for some imports
   commands:
     - pip check
   imports:
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:33]: Could you add `doc_source_url`: `https://github.com/joblib/joblib/tree/1.1.1/doc` ?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -21,14 +20,14 @@ requirements:
...
-    - python >=3.7
+    - python
     # Optional dependencies are numpy, python-lz4, psutil
 
 test:
   requires:
     - pip
-    - python <3.10
-    - pytest
+    - python
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:29]: `python` can be removed from `test/requires`

File: recipe/meta.yaml (hunk 1 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml:33]: And missing a `description`:
```
Joblib is a set of tools to provide lightweight pipelining in Python. In particular: transparent disk-caching of functions and lazy re-evaluation (memoize pattern), easy simple parallel computing.
Joblib is optimized to be fast and robust on large data in particular and has specific optimizations for numpy arrays.
```

**Review by danpetry — APPROVED** [review]

File: recipe/meta.yaml (hunk 2 above)
Comment (marcoesters) [reply recipe/meta.yaml]: Done, thanks!

**Review by varlackc — APPROVED** [review]
LGTM

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**